src/main/java/org/folio/rest/service/report/RefundReportService.java

```diff
@@ -464,11 +465,11 @@ private static String singleOrDefaultMessage(List<String> values, String message
   }
 
   private static String getStaffInfo(String comments) {
-    return parseFeeFineComments(comments).get(STAFF_COMMENTS_KEY);
+    return StringUtils.defaultString(parseFeeFineComments(StringUtils.defaultString(comments)).get(STAFF_COMMENTS_KEY));
```

Comment:
Maybe `parseFeeFineComments` should check if `comments` is null and return an empty map if it is? This way we can avoid second call of `defaultString` here. And please add static imports for utility methods where it makes sense.